apps/vscode/command_client/command_client.py

```diff
@@ -197,20 +198,30 @@ def run_command(
     return decoded_contents["returnValue"]
 
 
+command_server_directory = None
+
+
+# NB: See https://github.com/talonhub/community/issues/966 for why we do OS-specific temp dirs
+def get_platform_specific_communication_dir_path():
+    home_dir = Path(os.path.expanduser("~"))
+    directory_name = actions.user.command_server_directory()
+    if app.platform == "linux" or app.platform == "mac":
+        return home_dir / ".talon" / " .comms" / directory_name
+    elif app.platform == "windows":
+        # subprocess.run(["attrib","+H","myfile.txt"],check=True)
+        return home_dir / "AppData" / "Roaming" / "talon" / ".comms" / directory_name
```

Comment:
This also makes little sense on Windows where the dot prefix doesn't "do" anything.

Reply:
Originally I didn't have it prefixed I think, but it was switched for consistency of paths across all platforms.

Reply:
I don't think it matters much. But yeah the talon user path has no prefixed dot either so there is no consistency at that level anyway. So in this case i think actually using "comms" without dot on Windows is probably better? Happy either way.

Reply:
@nriley  Doesn't the dot prefix still hide a directory on windows in certain views?
Personally I would slightly prefer .comms everywhere for consistency if you compare the talon settings folder  on different systems -  even though of course the name and path of the talon settings folder will be widely different.

Reply:
> Doesn't the dot prefix still hide a directory on windows in certain views?

Nope, the hidden bit is separate metadata to the filename on Windows.

Reply:
I would personally vote for the consistency on this bit fwiw, as it is harmless.

Reply:
Will resolve this since knausj85 agrees, but feel free to open if you want to still debate.